models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -73,6 +73,7 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
   cpruncmd <- cpoutcmd <- rmoutdircmd <- rmrundircmd <- ""
   if (!is.null(settings$host$rabbitmq)) {
     #rsync cmd from remote to local host.
+    settings$host$rabbitmq$cpfcmd <- if(settings$host$rabbitmq$cpfcmd == NULL) "" else settings$host$rabbitmq$cpfcmd 
```

Comment:
@meetagrawal09 this syntax doesn't work in R. First, `if(settings$host$rabbitmq$cpfcmd == NULL)` throws an "argument is of length zero" error if the variable is NULL and should be replaced with is.null(). Second, the `if ... else` syntax should be replaced with the `ifelse()` function.

Reply:
Don't know but this operator did not work for me. Shows me the below error. 
```
Error in settings$host$rabbitmq$cpfcmd %||% "" :  could not find function "%||%"
```

Reply:
Oh, hm. 
(1) You'll need to import it[1] from rlang with `#' @importFrom rlang %||%` or just copy in the definition: `function (x, y) { if (is.null(x)) y else x}` ...
(2) ...which shows I was remembering wrong and `%||%` only takes scalar inputs, so my advice was bad to start with 🤦 

So `ifelse` it is!

[1]  (`%||%` has been added to base R in 4.4.0, so a few version bumps from now we'll be able to use it with no import)